src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -762,10 +788,18 @@ private string BuildSourcesString(HashSet<Tuple<string, string>> sources)
             var sb = new StringBuilder();
             foreach (Tuple<string, string> tuple in sources)
             {
-                if (tuple.Item1 == null || tuple.Item2 == null) { continue; }
+                if (sb.Length > 0) { sb.Append(", "); }
+
                 // Item1 is the name, Item2 is the source type, e.g., parameter
-                sb.Append(tuple.Item1 + "(" + tuple.Item2 + ")");
+                if (!string.IsNullOrWhiteSpace(tuple.Item1)) { sb.Append(tuple.Item1 + ": "); }
```

Comment:
>string.IsNullOrWhiteSpace [](start = 21, length = 25)

you probably don't need this check. verify by trying to execute this code (sb.Append(null + ":"). the compiled string.concat call handles null gracefully, i believe

Reply:
I don't want "" either.

---
In reply to: [282149859](https://github.com/microsoft/sarif-sdk/pull/1480#discussion_r282149859) [](ancestors = 282149859)